test/extensions/filters/network/dubbo_proxy/conn_manager_test.cc

```diff
@@ -1151,8 +1278,11 @@ TEST_F(ConnectionManagerTest, SendLocalReplyInMessageDecoded) {
         buffer.add(fake_response);
         return DubboFilters::DirectResponse::ResponseType::ErrorReply;
       }));
-  EXPECT_CALL(*decoder_filter_, onMessageDecoded(_, _))
+  EXPECT_CALL(*decoder_filter, onMessageDecoded(_, _))
       .WillOnce(Invoke([&](MessageMetadataSharedPtr, ContextSharedPtr) -> FilterStatus {
+        EXPECT_EQ(1, conn_manager_->getActiveMessagesForTesting().size());
+        EXPECT_NE(nullptr, conn_manager_->getActiveMessagesForTesting().front()->metadata());
+        conn_manager_->getActiveMessagesForTesting().front()->clearMetadataForTesting();
```

Comment:
why do you need clearMetadata here?

Reply:
sorry, i remove the clearMetadataForTesting function because there is no sense of validation.